Sources/MapboxNavigation/CarPlayNavigationViewController.swift

```diff
@@ -916,14 +918,8 @@ open class CarPlayNavigationViewController: UIViewController, BuildingHighlighti
             text += "\n\(secondaryText)"
         }
         primaryManeuver.instructionVariants = [text]
-        
-        // Add maneuver arrow
-        if #available(iOS 13.0, *) {
-            primaryManeuver.symbolImage = visualInstruction.primaryInstruction.maneuverImage(side: visualInstruction.drivingSide,
-                                                                                             type: styleManager?.currentStyleType)
-        } else {
-            primaryManeuver.symbolSet = visualInstruction.primaryInstruction.maneuverImageSet(side: visualInstruction.drivingSide)
-        }
+
+        primaryManeuver.symbolSet = visualInstruction.primaryInstruction.maneuverImageSet(side: visualInstruction.drivingSide)
```

Comment:
for history, some info on why we made this change.

From Apple CarPlay development guide: 
>  If the maneuver has an associated symbol, such as a turn right arrow, provide an image
using symbolSet. [...] You must provide two image variants using CPImageSet—one is used for rendering
the symbol on light backgrounds, the other is used for rendering the symbol on dark backgrounds. 

Since we don't know which custom background color a developer might provide, we always use `symbolSet` to pass both white and dark symbols for maneuvers, CarPlay will choose a better version on its own.